float32 image_map_index
sensor_msgs/Image image_map
sensor_msgs/Image image_camera
---
float32[] differences
float32[] histogram
